roles/openshift_node/templates/node.service.j2

```diff
@@ -8,6 +8,7 @@ Wants={{ openshift.docker.service_name }}.service
 Documentation=https://github.com/openshift/origin
 Requires=dnsmasq.service
 After=dnsmasq.service
+{% if openshift.docker.use_crio %}Wants=cri-o.service{% endif %}
```

Comment:
Originally we were keeping docker in there as ``use_crio`` would be additive. I'm OK with doing that once we know for sure docker is not needed for anything.